pkg/interfaces/contracts/vault/IVaultErrors.sol

```diff
@@ -289,61 +330,80 @@ interface IVaultErrors {
+     * @param underlyingToken The underlying token returned by `asset`
+     */
     error WrongUnderlyingToken(IERC4626 wrappedToken, address underlyingToken);
 
-    /// @dev A wrapped token reported the zero address as its underlying token asset.
-    error InvalidUnderlyingToken();
+    /**
+     * @notice A wrapped token reported the zero address as its underlying token asset.
+     * @dev This should never happen, but a malicious wrapper contract might do this (e.g., in an attempt to
+     * re-initialize the buffer).
+     *
+     * @param wrappedToken The wrapped token corresponding to the buffer
+     */
+    error InvalidUnderlyingToken(IERC4626 wrappedToken);
 
-    /// @dev The amount given to wrap/unwrap was too small, which can introduce rounding issues.
-    error WrapAmountTooSmall(address wrappedToken);
+    /**
+     * @notice The amount given to wrap/unwrap was too small, which can introduce rounding issues.
+     * @param wrappedToken The wrapped token corresponding to the buffer
+     */
+    error WrapAmountTooSmall(IERC4626 wrappedToken);
 
-    /// @dev Vault buffers are paused.
+    /// @notice Buffer operation attempted while vault buffers are paused.
     error VaultBuffersArePaused();
 
-    /// @dev Buffer shares were minted to an invalid address.
-    error BufferSharesInvalidReceiver(address to);
+    /// @notice Buffer shares were minted to the zero address.
+    error BufferSharesInvalidReceiver();
 
-    /// @dev Buffer shares were burnt from an invalid address.
-    error BufferSharesInvalidOwner(address from);
+    /// @notice Buffer shares were burned from the zero address.
+    error BufferSharesInvalidOwner();
 
     /*******************************************************************************
                                     Miscellaneous
     *******************************************************************************/
 
-    /// @dev Optional User Data should be empty in the current add / remove liquidity kind.
-    error UserDataNotSupported();
-
-    /// @dev Pool does not support adding / removing liquidity with an unbalanced input.
+    /// @notice Pool does not support adding / removing liquidity with an unbalanced input.
     error DoesNotSupportUnbalancedLiquidity();
 
-    /// @dev The contract should not receive ETH.
+    /// @notice The contract should not receive ETH.
     error CannotReceiveEth();
 
-    /// @dev The Vault extension was called by an account directly; it can only be called by the Vault via delegatecall.
+    /**
+     * @notice The Vault extension was called by an account directly.
+     * @dev It can only be called by the Vault via delegatecall.
+     */
     error NotVaultDelegateCall();
 
-    /// @dev Error thrown when a function is not supported.
-    error OperationNotSupported();
-
-    /// @dev The vault extension was configured with an incorrect Vault address.
+    /// @notice The vault extension was configured with an incorrect Vault address.
```

Comment:
```suggestion
    /// @notice The Vault extension was configured with an incorrect Vault address.
```

Not sure if we want to adopt capitalized Vault for the extension and admin, so this is just a suggestion.

Reply:
Will make it consistent (also found a copy/paste bug where it said VaultExtension and meant VaultAdmin)